platform/ext/target/nordic_nrf/common/core/startup.c

```diff
@@ -25,6 +25,27 @@
 #include "hw_init.h"
 #include "startup.h"
 
+#ifndef DOMAIN_NS
+#include "utilities.h"
+#endif
+
+void default_irq_handler(void)
+{
+	/*
+	 * This file is used by the TF-M build system. Usually in the
+	 * secure domain (tfm_s.elf), but also for testing purposes in the
+	 * non-secure domain (tfm_ns.elf).
+	 *
+	 * When used in the non-secure domain we do not have
+	 * tfm_core_panic so we loop forever instead.
+	 */
+#ifdef DOMAIN_NS
```

Comment:
I suspect you need this for BL2 as well? BL2 will not have tfm_core_panic and will have compilation error?

Reply:
Is that for vanilla TF-M?

NCS TF-M does not use BL2 right?

Reply:
Maybe BL2 is used for "our" MCUBoot integration?

Reply:
I was reviewing this patch in the context that you would put this upstream, so then you need the BL2 support.